src/lightning_lite/plugins/environments/slurm.py

```diff
@@ -160,7 +160,10 @@ def _validate_srun_used() -> None:
         """
         if _IS_WINDOWS:
             return
-        srun_exists = subprocess.call(["which", "srun"]) == 0
+        try:
+            srun_exists = subprocess.call(["command", "-v", "srun"]) == 0
+        except:
```

Comment:
```suggestion
        except FileNotFoundError:
```

we probably don't want to silence other errors